lib/post.ml

```diff
@@ -22,108 +22,30 @@ type t = {
   feed : Feed.t;
   author : string;
   email : string;
-  content : Nethtml.document list;
+  content : Soup.soup Soup.node;
   mutable link_response : (string, string) result option;
 }
 
-let rec len_prefix_of_html html len =
-  if len <= 0 then (0, [])
-  else
-    match html with
-    | [] -> (len, [])
-    | el :: tl ->
-        let len, prefix_el = len_prefix_of_el el len in
-        let len, prefix_tl = len_prefix_of_html tl len in
-        (len, prefix_el :: prefix_tl)
-
-and len_prefix_of_el el len =
-  match el with
-  | Nethtml.Data d ->
-      let len' = len - String.length d in
-      (len', if len' >= 0 then el else Data (String.sub d 0 len ^ "…"))
-  | Nethtml.Element (tag, args, content) ->
-      (* Remove "id" and "name" to avoid duplicate anchors with the whole
-         post. *)
-      let args = List.filter (fun (n, _) -> n <> "id" && n <> "name") args in
-      let len, prefix_content = len_prefix_of_html content len in
-      (len, Element (tag, args, prefix_content))
-
-let prefix_of_html html len = snd (len_prefix_of_html html len)
```

Comment:
These `prefix_of`... functions appeared to be dead code. Since they also referenced module `Nethtml`, I removed it.